app/views/content/life-as-a-teacher/teaching-as-a-career/skills-to-teach/_article.html.erb

```diff
@@ -0,0 +1,52 @@
+    <p>Teaching is a job that requires enthusiasm, subject passion, and a range of skills. Some skills you’ll learn through your teacher training and develop throughout your career. Other skills may be more natural or abilities you’ve developed through your life experience and previous jobs.</p>
+
+    <p>The skills needed to be a teacher will also vary by teaching role. For example, the skills needed for teaching secondary differ to the required skills to be a primary teacher.</p>
+
+    <p><%= render 'content/shared/quotes/quote_skills_ben' %></p>
+
+    <h2 class="heading--box-blue">Subject knowledge</h2>
+    <p>One of the most important skills for teaching is enthusiasm and knowledge of the subject you want to teach. This allows you to answer questions your pupils have and be creative in how you help them learn.</p>
+
+    <p>You can build your subject knowledge in several ways. For example, through a degree in a subject, your teacher training, or conversations with your colleagues and pupils. If you do postgraduate teacher training, your provider may decide that you'd benefit from <a href="/how-to-apply-for-teacher-training/subject-knowledge-enhancement">a subject knowledge enhancement course</a>.</p>
+
+    <p>Being passionate about building and sharing subject knowledge will help you inspire your pupils to learn and fulfil their potential.</p>
+
+    <p><%= render 'content/shared/quotes/quote_skills_mark' %></p>
+
+    <h2 class="heading--box-blue">Pedagogy</h2>
+    <p>Pedagogy is the theory and practices of teaching. In your teacher training, you’ll learn all the various ways you can teach. You’ll be equipped with a toolkit to put into practice during your teacher training placements and throughout your career.</p>
+
+    <p><%= render 'content/shared/quotes/quote_skills_jane' %></p>
+
+    <h2 class="heading--box-blue">Soft skills</h2>
+    <p>As teaching is a very people-focused career, your soft skills are important. These are natural attributes you develop through your life experience and previous jobs.</p>
+
+    <p>Soft skills that are useful as a teacher include:</p>
+
+    <ul>
+      <li>adaptability</li>
+      <li>communication</li>
+      <li>creativity</li>
+      <li>empathy</li>
+      <li>organisation</li>
+      <li>patience</li>
+      <li>problem solving</li>
+      <li>resilience</li>
+      <li>teamwork</li>
+    </ul>
+
+    <p>These skills will help you build relationships with your pupils, colleagues, and parents. They’ll also help you manage your workload and adapt to the changing needs of your pupils and school.</p>
+
+    <p><%= render 'content/shared/quotes/quote_skills_jodie' %></p>
+
+    <h2 class="heading--box-blue">Find out more about the skills needed to become a teacher</h2>
+    <p>To find out more about what skills a teacher needs, you can attend a free <a href="/events">Get Into Teaching event</a> or <a href="/teacher-training-advisers">get a teacher training adviser</a> if you’re eligible.</p>
```

Comment:
Whilst I quite like the next steps (including events) we now have 2 adviser links next to each other (here and in the promo) with different link text going to the same destination - not great for accessibility!

Do we need this section? Should we just have the adviser promo?